colorpair_test.cpp

```diff
@@ -0,0 +1,45 @@
+#include "colorpair.h"
+#include "colorutils.h"
+
+#include <iostream>
+#include <assert.h>
+
+void testNumberToPair(int pairNumber,
+	TelCoColorCoder::MajorColor expectedMajor,
+	TelCoColorCoder::MinorColor expectedMinor)
+{
+	TelCoColorCoder::ColorPair colorPair =
+		TelCoColorCoder::GetColorFromPairNumber(pairNumber);
+	std::cout << "Got pair " << colorPair.ToString() << std::endl;
+	assert(colorPair.getMajor() == expectedMajor);
+	assert(colorPair.getMinor() == expectedMinor);
+}
+
+void testPairToNumber(
+	TelCoColorCoder::MajorColor major,
+	TelCoColorCoder::MinorColor minor,
+	int expectedPairNumber)
+{
+	int pairNumber = TelCoColorCoder::GetPairNumberFromColor(major, minor);
+	std::cout << "Got pair number " << pairNumber << std::endl;
+	assert(pairNumber == expectedPairNumber);
+}
+
+void testFormattedColorCoding()
+{
+	std::string formattedColorCoding = TelCoColorCoder::GetFormattedColorCoding();
+	std::cout << "Formatted Color Coding : \n" << formattedColorCoding << std::endl;
```

Comment:
This still requires manual inspection. Think of an assert to reduce the manual burden?